src/App.tsx

```diff
@@ -1,24 +1,187 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Please remove the unnecessary eslint-disable rule for 'jsx-a11y/control-has-associated-label' as it is not required in this file.